src/chat/Chat.js

```diff
@@ -9,12 +9,28 @@ import MessageList from '../message-list/MessageList';
 import NoMessages from '../message/NoMessages';
 import ComposeBox from '../compose/ComposeBox';
 
-export default class MainScreen extends React.Component {
+export default class Chat extends React.Component {
+
+  unsentMessageIds = [];
```

Comment:
Yeah, we have this problem not resolved completely.
I'll create an issue for some tests on this.